packages/config/config/devices/0x027a/zse44.json

```diff
@@ -0,0 +1,363 @@
+					"value": 2
+				},
+				{
+					"label": "0xff (on) to Lifeline and Group 3",
+					"value": 3
+				},
+				{
+					"label": "0x00 (off) to Group 3 only",
+					"value": 4
+				},
+				{
+					"label": "0x00 (off) to Lifeline and Group 3",
+					"value": 5
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Group 3 only",
+					"value": 6
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Lifeline and Group 3",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "9",
+			"label": "High Humidity Alert Threshold",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "10",
+			"label": "High Humidity Alert Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 7,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Lifeline only",
+					"value": 1
+				},
+				{
+					"label": "0xff (on) to Group 4 only",
+					"value": 2
+				},
+				{
+					"label": "0xff (on) to Lifeline and Group 4",
+					"value": 3
+				},
+				{
+					"label": "0x00 (off) to Group 4 only",
+					"value": 4
+				},
+				{
+					"label": "0x00 (off) to Lifeline and Group 4",
+					"value": 5
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Group 4 only",
+					"value": 6
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Lifeline and Group 4",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Low Humidity Alert Threshold",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 100,
+			"defaultValue": 0,
+			"unsigned": true
+		},
+		{
+			"#": "12",
+			"label": "Low Humidity Alert Reporting",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 7,
+			"defaultValue": 7,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Lifeline only",
+					"value": 1
+				},
+				{
+					"label": "0xff (on) to Group 5 only",
+					"value": 2
+				},
+				{
+					"label": "0xff (on) to Lifeline and Group 5",
+					"value": 3
+				},
+				{
+					"label": "0x00 (off) to Group 5 only",
+					"value": 4
+				},
+				{
+					"label": "0x00 (off) to Lifeline and Group 5",
+					"value": 5
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Group 5 only",
+					"value": 6
+				},
+				{
+					"label": "0xff (on) and 0x00 (off) to Lifeline and Group 5",
+					"value": 7
+				}
+			]
+		},
+		{
+			"#": "13",
+			"label": "Temperature Scale",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celsius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "14",
+			"label": "Temperature Offset - 0=-10, 100=0, 200=+10",
```

Comment:
This is a particularly screwy parameter, but put that information in the description not the label.